server/middleware/session/setSession.js

```diff
@@ -74,6 +74,11 @@ const setSession = async (req, res, next) => {
     user = await jwtVerify(idToken)
   } catch (e) {
     // console.error('Jwt Verify failed', e)
+
+    if (e.name === 'TokenExpiredError') {
+      res.clearCookie('idToken')
```

Comment:
it might be safer to use unsetToken from auth.js.

Reply:
I was under the impression that unsetToken was only for use in browser (because it uses `js-cookie`) where as this code is always running on the server side